assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -480,6 +480,12 @@ che.workspace.plugin_registry_url=https://che-plugin-registry.prod-preview.opens
 # In case Che plugins tooling is not needed value 'NULL' should be used
 che.workspace.devfile_registry_url=https://che-devfile-registry.prod-preview.openshift.io/
 
+# Defines a default behaviour if user does not configure persist volumes.
+# Possible values: true or false
+# In case of false - PersistentVolumeClaims are used by declared volumes by user and plugins.
+# In case of true - emptyDir is used instead of PVCs. Note that data will be lost after workspace restart.
+che.workspace.persist_volumes=true
```

Comment:
React with to vote for :1st_place_medal: `che.workspace.persist_volumes`
Or :2nd_place_medal:  for `che.workspace.persist_volumes.default`
Or leave your proposal here

Reply:
:2nd_place_medal: 
And I would expect `true` and `false` exactly opposite meaning. Or do I miss something?